README.md

```diff
@@ -30,6 +30,8 @@ The Learning Path Index is a dynamic and versatile repository designed to empowe
 
 ## [5. Keyword Extraction with KeyBERT and WordWise-Kaggle Notebook](https://github.com/neomatrix369/learning-path-index/blob/main/app/Keyword%20Extraction%20with%20KeyBERT%20and%20WordWise.ipynb)
 
+## [6. Learning Pathway Index Data Cleaning and Preprocessing](https://www.kaggle.com/code/manishkr1754/lpi-data-cleaning-and-preprocessing/notebook)
```

Comment:
Please also refer to [entzyeung](https://github.com/entzyeung)'s noebook 
 on Kaggle (see the one he refers to https://github.com/neomatrix369/learning-path-index/pull/14) - should be a hyper link to the Kaggle Notebook

Reply:
He has completed his work on his local machine rather than using a Kaggle notebook. I have provided him with Editor access to my Kaggle notebook enabling him to add his contributions. Nevertheless, I have already covered the majority of the common tasks, with the exception of Keyword Preprocessing, which I believe may not be necessary.

Reply:
In that case please refer to the notebook in the closed PR, see [notebook](https://github.com/entzyeung/learning-path-index/blob/5d0593ffc5bcf5e09e21e5f58f97932966e82250/label_clean_up.ipynb)

Reply:
> In that case please refer to the notebook in the closed PR, see [notebook](https://github.com/entzyeung/learning-path-index/blob/5d0593ffc5bcf5e09e21e5f58f97932966e82250/label_clean_up.ipynb)

I have referred to Lorentz notebook then I worked on my notebook and it includes all the tasks that is needed and I have shared with Lorentz as well.

Reply:
OK, Updated Data Info as per request